spec/units/commands/git_command_spec.rb

```diff
@@ -53,5 +59,55 @@
         env,
       )
     end
+
+    context 'with autocorrect enabled' do
```

Comment:
Is this new functionality? Or, I guess, why is this test here? Was this handled in the parser before?